src/xdsl/printer.py

```diff
@@ -311,6 +317,25 @@ def print_op_with_default_format(self, op: Operation) -> None:
         self._print_op_attributes(op.attributes)
         self.print_regions(op.regions)
 
+        if self.print_operand_types == Printer.TypeLocation.AFTER and self.print_result_types == Printer.TypeLocation.AFTER:
+            self.print(" : (")
+            first = True
+            for operand in op.operands:
```

Comment:
```suggestion
            self.print_list(op.operands, lambda attr: self.print_attribute(attr.typ))
```
to make this a bit more concise